crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -357,65 +357,66 @@ impl<DB: Database, C: Consensus, EF: ExecutorFactory> BlockchainTree<DB, C, EF>
         // TODO save pending block to database
```

Comment:
what about this one? and the comment above

Reply:
idk, not familiar with the context